### ISSUE TITLE
Usage issues + proposal

### ISSUE THREAD

**Issue by thomas-bc** [capsulecorplab/fprime-workspace-image#32]
Hey @capsulecorplab !

Question, I haven't been able to spin that up locally unfortunately. Here are my logs and a screenshot of what I get when trying to access the workspace in Google Chrome or Firefox. I'm running on an Intel Mac if that matters.

```
docker-compose up
Creating fprime-workspace-image_kasm_1 ... done
Attaching to fprime-workspace-image_kasm_1
kasm_1  | + DEFAULT_PROFILE_HOME=/home/kasm-default-profile
kasm_1  | + PROFILE_SYNC_DIR=/kasm_profile_sync
kasm_1  | + '[' -f /home/kasm-user/.bashrc ']'
kasm_1  | + echo 'Profile Sync Directory Does Not Exist. No Sync will occur'
kasm_1  | Profile Sync Directory Does Not Exist. No Sync will occur
kasm_1  | + copy_default_profile_to_home
kasm_1  | + echo 'Copying default profile to home directory'
kasm_1  | Copying default profile to home directory
kasm_1  | + cp -rp /home/kasm-default-profile/. /home/kasm-user/

```

![image](https://github.com/capsulecorplab/fprime-workspace-image/assets/49786685/83b1b052-468a-4313-bf76-583d7a29807b)

When going through http, I get `The connection was reset`. I quickly browsed through https://kasmweb.com/docs/latest/guide/troubleshooting.html but no luck.
Would you have some insights as to what may be going on?

Other question: if we're able to confirm it works as expected, would you be willing to consider migrating this repository over to https://github.com/fprime-community ? I'm thinking it could help this get some traction. Just an idea, totally your call :)

**Comment by capsulecorplab** [comment]
If you're running it for the first time, the `cp -rp /home/kasm-default-profile/. /home/kasm-user/` line will take a minute because it's creating the persistence layer. Once it gets past that, you should be able to access the image on https://localhost:6901in a web browser

**Comment by capsulecorplab** [comment]
If you're running it for the first time, the `cp -rp /home/kasm-default-profile/. /home/kasm-user/` line will take a minute because it's creating the persistence layer. Once it gets past that, you should be able to access the image on https://localhost:6901in a web browser 

And yes, I'd be more than happy to migrate this to the fprime community GitHub organization!

**Comment by thomas-bc** [comment]
@capsulecorplab ah damn you are right, I waited for a while but nothing was moving. Then saw your comment a while later, went back to the terminal and things had started up! My bad 
This is really nice!

**Comment by capsulecorplab** [comment]
@thomas-bc Awesome! Feel free to open any other issues as you try it out

**Comment by capsulecorplab** [comment]
Closing issue, as it's been resolved, but created #33 to continue discussion on migrating repo to fprime-community
